docs/source/notebooks/Image_Tut.ipynb

```diff
@@ -0,0 +1,468 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.datasets import fetch_openml\n",
+    "\n",
+    "# Fetch the MNIST dataset\n",
+    "mnist = fetch_openml('mnist_784')\n",
+    "\n",
+    "# Assign the data features and labels to variable X and y respectively \n",
+    "X = mnist.data.astype('float32')\n",
+    "y = mnist.target.astype('int64')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Bringing Your Own Data (BYOD)?\n",
+    "\n",
+    "Assign your data's features to variable ``X`` and its labels to variable ``y`` instead.\n",
+    "\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "# Scale the features which range from 0 to 255 (256 levels on the grayscale) \n",
+    "X /= 255.0\n",
+    "\n",
+    "# Then convert it to a numpy array\n",
+    "X = X.to_numpy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Define a classification model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, we define a simple neural network with PyTorch."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from torch import nn\n",
+    "\n",
+    "class NeuralNetwork(nn.Module):\n",
+    "    def __init__(self):\n",
+    "        super(NeuralNetwork, self).__init__()\n",
+    "        self.dropout = nn.Dropout(0.5)\n",
+    "        self.hidden = nn.Linear(28*28, 128)\n",
+    "        self.output = nn.Linear(128, 10)\n",
+    "\n",
+    "    def forward(self, X):\n",
+    "        X = nn.functional.relu(self.hidden(X))\n",
+    "        X = self.dropout(X)\n",
+    "        X = nn.functional.softmax(self.output(X), dim=-1)\n",
+    "        return X"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Wrap with Skorch for scikit-learn compatibility"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As some of Cleanlab's feature requires scikit-learn compatibility, we will need to adapt the above PyTorch neural net accordingly. Skorch is a convenient package that helps with this, read more about it here: https://skorch.readthedocs.io"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from skorch import NeuralNetClassifier\n",
+    "\n",
+    "# Wrap the PyTorch neural net around Skorch's NeuralNetClassifier wrapper\n",
+    "net = NeuralNetClassifier(NeuralNetwork)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Compute the cross-validated predicted probabilities"
```

Comment:
I'd rename "cross-validated predicted probabilities" -> "out-of-sample predicted class probabilities".

Cross-validation is just one way to get out-of-sample predicted probabilities for each datapoint, the latter is a more general idea (eg. cross-validation is not needed if the user only wants to find label errors in held-out test data)

Reply:
done